rocAL/rocAL/source/master_graph.cpp

```diff
@@ -400,6 +401,24 @@ MasterGraph::sequence_frame_timestamps(std::vector<std::vector<float>> &sequence
     _sequence_frame_timestamps_vec.pop_back();
 }
 
+std::vector<uint32_t>
```

Comment:
The returned vector is popped from the back of _resize_width and stored in a local variable. So I don't think it can be passed as a reference. Leaving it as it is in the latest commit.